test-suite/Network/CloudSeeder/MainSpec.hs

```diff
@@ -517,3 +517,62 @@ spec =
                 rootExpectedTags
                 :-> "csid"
             , RunChangeSet "csid" :-> () ]
+
+    context "global stacks" $ do
+      it "global stacks can be deployed into the global environment" $ do
+        let config' = deployment "foo" $ do
+              stack "repo"
+                global
+              stack_ "base"
+        runSuccess $ cli config'
+          & stubFileSystemT
+            [ ("repo.yaml", rootTemplate) ]
+          & stubEnvironmentT []
+          & stubCommandLineT ["provision", "repo", "global"]
+          & stubExceptT
+          & mockCloudT
+            [ GetStackInfo "global-foo-repo" :-> Nothing
+            , ComputeChangeset
+                "global-foo-repo"
+                CreateStack
+                rootTemplate
+                [("Env", Value "global")]
+                [("cj:application", "foo"), ("cj:environment", "global")]
+                :-> "csid"
+            , RunChangeSet "csid" :-> () ]
+
+      it "other stacks can not be deployed into the global namespace" $ do
```

Comment:
Should there be a test that is the inverse of this one? That is, global stacks shouldn’t be allowed to be deployed into any namespace *other* than the global one.